src/test/kotlin/org/jmailen/gradle/kotlinter/functional/KotlinJsProjectTest.kt

```diff
@@ -0,0 +1,115 @@
+package org.jmailen.gradle.kotlinter.functional
+
+import org.gradle.testkit.runner.TaskOutcome
+import org.jmailen.gradle.kotlinter.functional.utils.kotlinClass
+import org.jmailen.gradle.kotlinter.functional.utils.resolve
+import org.jmailen.gradle.kotlinter.functional.utils.settingsFile
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import java.io.File
+
+class KotlinJsProjectTest : WithGradleTest.Kotlin() {
+
+    lateinit var projectRoot: File
+
+    @BeforeEach
+    fun setup() {
+        projectRoot = testProjectDir.apply {
+            resolve("settings.gradle") { writeText(settingsFile) }
+            resolve("build.gradle") {
+                // language=groovy
+                writeText(
+                    """
+                    plugins {
+                        id 'org.jetbrains.kotlin.js'
+                        id 'org.jmailen.kotlinter'
+                    }
+                    
+                    repositories.mavenCentral()
+                    
+                    kotlin {
+                        js {
+                            browser()
+                            binaries.executable()
+                        }
+                    }
```

Comment:
I have literally zero experience with JS, so the test environment only follows the [official documentation](https://kotlinlang.org/docs/js-project-setup.html#execution-environments)